include/behaviortree_cpp_v3/decorators/delay_node.h

```diff
@@ -38,18 +39,27 @@ class DelayNode : public DecoratorNode
         return {InputPort<unsigned>("delay_msec", "Tick the child after a few milliseconds")};
     }
 
+    void halt() override { 
+      
+     timer_.cancelAll();
+     DecoratorNode::halt();
+     
+      }
```

Comment:
maybe remove the spaces and extra line here? So that `}` aligns with `void`

Reply:
Done